src/DssSpell.sol

```diff
@@ -60,44 +42,20 @@ contract DssSpellAction is DssAction {
-    // NOTE: previous line will be different on the mainnet since the chains are out of sync
-    // address internal immutable RWA015_A_OUTPUT_CONDUIT_USDC = DssExecLib.getChangelogAddress("RWA015_A_OUTPUT_CONDUIT_LEGACY");
-    address internal immutable RWA015_A_URN                = DssExecLib.getChangelogAddress("RWA015_A_URN");
-    address internal immutable RWA015_A_OUTPUT_CONDUIT_PAX = DssExecLib.getChangelogAddress("RWA015_A_OUTPUT_CONDUIT");
-    address internal immutable MCD_PSM_PAX_A               = DssExecLib.getChangelogAddress("MCD_PSM_PAX_A");
-    address internal immutable MCD_PSM_GUSD_A              = DssExecLib.getChangelogAddress("MCD_PSM_GUSD_A");
-    address internal immutable MCD_PSM_USDC_A              = DssExecLib.getChangelogAddress("MCD_PSM_USDC_A");
-    address internal immutable MCD_ESM                     = DssExecLib.esm();
+    // ----- JAT1 DAO Resolution -----
+    // Forum: https://forum.makerdao.com/t/clydesdale-quarterly-return-of-surplus-fund/21291
+    // Poll: N/A
+    // Approve DAO Resolution hash QmaGTVioBsCPfNoz9rbW7LU6YuzfgqHDZd92Hny5ACfL3p
+
+    // Comma-separated list of DAO resolutions IPFS hashes.
+    string public constant dao_resolutions = "QmaGTVioBsCPfNoz9rbW7LU6YuzfgqHDZd92Hny5ACfL3p";
+
+    address internal immutable MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
 
     // Spark
-    address internal immutable SUBPROXY_SPARK              = DssExecLib.getChangelogAddress("SUBPROXY_SPARK");
-    // NOTE: goerli spell address is originated from https://github.com/marsfoundation/spark-spells/pull/9
-    address internal constant SPARK_SPELL                  = 0x2Ad00613A66D71Ff2B0607fB3C4632C47a50DADe;
-
-    // CRON jobs
-    // NOTE: ignore in goerli
-    // address internal constant CRON_SEQUENCER            = ;
-    // address internal constant CRON_AUTOLINE_JOB         = ;
-    // address internal constant CRON_LERP_JOB             = ;
-    // address internal constant CRON_D3M_JOB              = ;
-    // address internal constant CRON_CLIPPER_MOM_JOB      = ;
-    // address internal constant CRON_ORACLE_JOB           = ;
-    // address internal constant CRON_FLAP_JOB             = ;
-
-    // New flapper
-    address internal constant MCD_FLAP                     = 0x584491031764f94a97a0f98bBe536B004Ab9467b;
-    address internal constant FLAPPER_MOM                  = 0x7316C080BFd1c8857605627a251A2F0ae511E4A1;
-    address internal constant PIP_MKR                      = 0x496C851B2A9567DfEeE0ACBf04365F3ba00Eb8dC;
+    address internal immutable SUBPROXY_SPARK = DssExecLib.getChangelogAddress("SUBPROXY_SPARK");
+    // NOTE: goerli spell address is originated from https://github.com/marsfoundation/spark-spells/blob/d41d58ccc974f8358b0df962ad1fb931fedb7e62/src/proposals/20230802/SparkGoerli_20230802.t.sol#L80
+    address internal constant SPARK_SPELL     = 0xEd3BF79737d3A469A29a7114cA1084e8340a2f20;
```

Comment:
the deployer of this spell is `0x02416B99202081F6b90851e35682Ca90D547054c` ([tx](https://goerli.etherscan.io/tx/0xe933b3e4f43a0a887f23476259bc31877a8b7aaf12e779db103e4458aa43cd35))
and is not in the `addresses_deployers`. Should it be added?

Reply:
Should we care about who deploys the Spark spell?

I'm currently deep diving into the spell test suite, and the only time the addresses in `addresses_deployers.sol` are used is in the [`checkAuth` test](https://github.com/makerdao/spells-mainnet/blob/c1f3caa1374e4f911db6a246852947c2b76692b3/src/DssSpell.t.base.sol#L1529-L1545).

This test basically fetches all contracts in the chainlog and checks if all deployers have been `deny`ed in those.

Spells don't make it into the chainlog, so having the deployer of a spell there does not do anything other than maybe keeping a list there. This might be important for MakerDAO spells, for the sake of the spell-crafting process, but I don't think we should have Spark spells deployers there.

Reply:
Ok. valid point if it is indeed used only there. I see you've left the [comment](https://github.com/makerdao/pe-checklists/pull/12/files#r1281821904) already so here the outcome should be according to resolution there.

Reply:
I added the deployer as a comment to that file.